fenix/app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -670,6 +670,9 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
                     subscribeForNewAddonsIfNeeded(components.supportedAddonsChecker, extensions)
                 },
                 onUpdatePermissionRequest = components.addonUpdater::onUpdatePermissionRequest,
+                onDisabledExtensionProcessSpawning = {
+                    // https://bugzilla.mozilla.org/show_bug.cgi?id=1846979
+                },
```

Comment:
Nit: As we are not providing any functionality here, maybe we could add this as part of [Bug 1846979](https://bugzilla.mozilla.org/show_bug.cgi?id=1846979). So that patch will contain everything related to its behaviours.

Reply:
agreed and also no longer needed after [PR 3472](https://github.com/mozilla-mobile/firefox-android/pull/3472) 😄